lib/src/components/range_selector/range_selector.dart

```diff
@@ -0,0 +1,455 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import '../../../zeta_flutter.dart';
+
+/// The [ZetaRangeSelector] is a customizable range selector widget that
+/// allows users to select a range of values within a specified minimum
+/// and maximum range. It provides a visual representation of the selected
+/// range and allows for precise adjustments through both dragging and direct input
+class ZetaRangeSelector extends ZetaStatefulWidget {
+  /// Creates a new [ZetaRangeSelector]
+  ZetaRangeSelector({
+    super.key,
+    super.rounded,
+    required this.initialValues,
+    this.min = 0.0,
+    this.max = 100,
+    this.label,
+    this.onChange,
+    this.divisions,
+    this.semanticLabel,
+    this.showValues = true,
+    this.largeSliderThumbs = false,
```

Comment:
I can't see the large thumbs in the designs - can we remove this until that is added there please